flink-contrib/docker-flink/Dockerfile

```diff
@@ -22,7 +22,7 @@ FROM java:8-jre-alpine
 RUN apk add --no-cache bash snappy
 
 # Configure Flink version
-ARG FLINK_VERSION=1.1.3
+ARG FLINK_VERSION=1.1.4
```

Comment:
The main reason for the upgrade that I needed was the the mirrors did not have version 1.1.3. They only had 1.1.4 and 1.2.0. Which is probably a different issue.

Example: http://mirrors.advancedhosters.com/apache/flink/